cluster/aws/config-default.sh

```diff
@@ -38,6 +38,7 @@ PORTAL_NET="10.0.0.0/16"
 
 # Optional: Install node monitoring.
 ENABLE_NODE_MONITORING=true
+ELASTICSEARCH_LOGGING_REPLICAS=1
```

Comment:
Should probably be lumped with other logging flags, rather than with node-monitoring